### PR TITLE
erc20 factory interface

### PR DESCRIPTION
**Pull request by Flip-Liquid**

### CODE REVIEW

File: src/interfaces/IValoremERC20Factory.sol [hunk 1]
```diff
@@ -20,5 +20,75 @@ pragma solidity 0.8.11;
...
+    /*//////////////////////////////////////////////////////////////
+    //  Accessors
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice Retrieves the wrapper ERC20 contract address for a supplied Valorem core token ID.
+     * @param tokenId The token ID for which to retrieve the wrapper.
+     * @return The address of the ERC20 wrapper contract.
+     */
+    function wrappers(uint256 tokenId) external returns (address);
```
Comment (0xAlcibiades) [inline src/interfaces/IValoremERC20Factory.sol:56]: should be singular

File: src/interfaces/IValoremERC20Factory.sol [hunk 2]
```diff
@@ -20,5 +20,75 @@ pragma solidity 0.8.11;
...
+
+    /*//////////////////////////////////////////////////////////////
+    //  Protocol Admin
+    //////////////////////////////////////////////////////////////*/
+
+    /// TODO: feeTo?
```
Comment (0xAlcibiades) [inline src/interfaces/IValoremERC20Factory.sol:62]: Nah

File: src/interfaces/IValoremERC20Factory.sol [hunk 3]
```diff
@@ -20,5 +20,75 @@ pragma solidity 0.8.11;
...
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice Deploy a new wrapper ERC20 contract for a given token ID.
+     * @param tokenId The Valorem core token ID for which to deploy a wrapper.
+     * @param name The name of the new ERC20 wrapper.
+     * @param symbol The symbol of the new ERC20 wrapper.
+     * @return The wrapper ERC20 address.
+     */
+    function newWrapperContract(
```
Comment (0xAlcibiades) [inline src/interfaces/IValoremERC20Factory.sol:75]: Reverts if it already exists?

File: src/interfaces/IValoremERC20Wrapper.sol [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+     * @param _amount Amount of tokens to be wrapped. 
+     */
+    function unwrap(address _account, uint256 _amount) external;
+
+    /**
+     * @notice Redeems an account's fractionalized share of an options lot for the underlying
+     * and exercise assets.
+     * @param _account The account for which the ERC20 balance is being redeemed.
+     */
+    function redeem(address _account) external;
```
Comment (0xAlcibiades) [inline src/interfaces/IValoremERC20Wrapper.sol:40]: These may complicate more than we need, although have definite ergonomic benefits, let's see in the implementation.

**Review by 0xAlcibiades — CHANGES_REQUESTED** [review]

File: src/interfaces/IValoremERC20Wrapper.sol (hunk 4 above)
Comment (Flip-Liquid) [reply src/interfaces/IValoremERC20Wrapper.sol:40]: Would you rather this be removed from the interface?

**Pull request merged**